plugins/org.jkiss.dbeaver.ext.hive/plugin.xml

```diff
@@ -40,9 +40,8 @@
                         categories="hadoop">
                     <replace provider="generic" driver="apache_hive2"/>
 
-                    <file type="jar"
-                          path="https://github.com/timveil/hive-jdbc-uber-jar/releases/download/v1.9-2.6.5/hive-jdbc-uber-2.6.5.0-292.jar"
-                          bundle="!drivers.hive"/>
+                    <file type="jar" path="maven:/org.apache.hive:hive-jdbc:RELEASE[4.0.0-standalone]" bundle="!drivers.hive"/>
```

Comment:
- To be honest I can't find any explanation in the dbeaver documentation on how to define the `classifier` property in this place. Should this be `maven:/org.apache.hive:hive-jdbc:RELEASE[4.0.0-standalone]`, or `maven:/org.apache.hive:hive-jdbc:standalone:RELEASE[4.0.0]`?

Reply:
No need to add `-standalone` here. Please left just `maven:/org.apache.hive:hive-jdbc:RELEASE[4.0.0`